search_config.admin.inc

```diff
@@ -7,7 +7,7 @@
 use Drupal\Core\Session\AccountInterface;
 use Drupal\Core\Url;
 function _search_config_form_search_admin_settings_alter($form, $form_state) {
-    $string_overrides = search_config_string_overrides();
+    $string_overrides = Drupal\module\search_config_string_overrides();
```

Comment:
Drupal\module is not needed b/c the function is in .module file. Same at other places.